frontend/src/layout/header/index.tsx

```diff
@@ -47,6 +53,7 @@ const SearchMode: React.FC = () => {
 
     return <>
         <ActionIcon title={t("back")} onClick={() => menu.close()} >
+            {/* TODO: Adjust color. */}
```

Comment:
Sorry for these. I figured it might come in handy later on, when we update the overall color scheme. But I wasn't really consistent with leaving the TODOs, so they may as be removed again. We'll need to go over everything anyway when doing the repaint (if that is even on the agenda anymore).